docs/fastdds/dds_layer/domain/domainParticipant/domainParticipant.rst

```diff
@@ -1,5 +1,167 @@
++--------------------------+------------------------------+----------+
+| ``entity_factory_``      | EntityFactoryQosPolicy       | yes      |
++--------------------------+------------------------------+----------+
+| ``allocation_``          | ParticipantResourceLimitsQos | no       |
++--------------------------+------------------------------+----------+
+| ``properties_``          | PropertyPolicyQos            | no       |
++--------------------------+------------------------------+----------+
+| ``wire_protocol_``       | WireProtocolConfigQos        | no       |
++--------------------------+------------------------------+----------+
+| ``transport_``           | TransportConfigQos           | no       |
++--------------------------+------------------------------+----------+
+
+The QoS value of a previously created DomainParticipant can be modified using the :func:`set_participant_qos()` method.
+Trying to modify an immutable QosPolicy on an already enabled DomainParticipant will result on an error.
+In such case, no changes will be applied and the DomainParticipant will keep its previous DomainParticipantQos.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+   :language: c++
+   :start-after: //DDS_CHANGE_DOMAINPARTICIPANTQOS
+   :end-before: //!
+
+
+.. _dds_layer_defaultDomainParticipantQos:
+
+Default DomainParticipantQos
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default DomainParticipantQos is the QoS definition that will be used when no other
+:ref:`dds_layer_defaultDomainParticipantQos` is provided to the DomainParticipant, either during
+creation of through the :func:`set_participant_qos()` method.
+
+The current default DomainParticipantQos can be retrieved using the :func:`get_default_participant_qos()` method
+on the :ref:`dds_layer_domainParticipantFactory` singleton.
+It can also be modified at any time using the :func:`set_default_participant_qos()` method
+on the :ref:`dds_layer_domainParticipantFactory` singleton.
+Once modified, all new instances of DomainParticipant created without specifying a
+:ref:`dds_layer_defaultDomainParticipantQos` value will use the new default value.
+DomainParticipant instances created before the modification will not be affected.
+
+The special symbol :class:`PARTICIPANT_QOS_DEFAULT` can be used as QoS parameter on :func:`create_participant()`
+or :func:`set_participant_qos()` methods to indicate that the current default DomainParticipantQos should be used.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+   :language: c++
+   :start-after: //DDS_CHANGE_DEFAULT_DOMAINPARTICIPANTQOS
+   :end-before: //!
+
+:func:`set_default_participant_qos()` method also accepts the symbol :class:`PARTICIPANT_QOS_DEFAULT`
+as input parameter.
+This will reset the current default DomainParticipantQos to default constructed values.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+   :language: c++
+   :start-after: //DDS_CHANGE_DOMAINPARTICIPANTQOS_TO_DEFAULT
+   :end-before: //!
```

Comment:
QoS value classes/structs are shared among different Entities. For example,  `EntityFactoryQosPolicy` is common for `DomainParticipantQos`, `PublisherQos` and `SubscriberQos`. Instead of defining the default values on each, the idea I had is to convert the class name to a reference to the corresponding section where that QoS is detailed. And the default value given.

But that needs the Core section to be done.

I'll add a comment to let the reader know where to go (and click once the ref is in place) to find that info.